frontend/packages/console-app/src/components/pdb/AvailabilityRequirement.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { useTranslation } from 'react-i18next';
+import { PodDisruptionBudgetKind } from './types';
+
+const AvailabilityRequirement: React.FC<AvailabilityRequirementProps> = ({ pdb, replicas }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {!_.isNil(pdb?.spec?.minAvailable)
+        ? t('console-app~Min available {{minAvailable}} of {{count}} pod', {
+            minAvailable: pdb.spec.minAvailable,
+            count: replicas,
+          })
+        : t('public~Max unavailable {{maxUnavailable}} of {{count}} pod', {
+            maxUnavailable: pdb?.spec?.maxUnavailable,
+            count: replicas,
+          })}
```

Comment:
Eg: 
![image](https://user-images.githubusercontent.com/8126518/164539144-438b3833-d597-4909-be93-9d6e6ed158a2.png)

Reply:
> `Max unavailable undefined of 1 pod` 🤔

@andrewballantyne I removed it since the value is not set in the API and it wouldn't impact the disruption. I think we should hide the `AvailabilityRequirement` field if the requirement is not set. Alternatively, we can set it to dash `-`  but it looks odd - Max unavailable - of 3 pods.

Reply:
Yeah, don't set it to dash... those are for full fields not for a partial replacement in a string. I recommend dropping it if it does not have enough data to render properly.